### PR TITLE
CODENVY-583: Fix JavaDoc feature

### PR DESCRIPTION
**Pull request by akorneta**
This pull request provides implementation for url modifier that adds machine token as a query parameter into specified url.
@skabashnyuk, @vparfonov

### CODE REVIEW

File: plugins/plugin-java/codenvy-plugin-java-ext-lang-server/src/main/java/com/codenvy/plugin/java/server/WsAgentURLProvider.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                              HttpJsonRequestFactory requestFactory) {
+        this.wsId = wsId;
+        this.workspaceApiEndpoint = apiEndpoint + "/workspace/";
+        this.requestFactory = requestFactory;
+    }
+
+    @Override
+    public String get() {
+        try {
+            final Collection<ServerDto> servers = requestFactory.fromUrl(workspaceApiEndpoint + wsId)
```
Comment (skabashnyuk) [inline plugins/plugin-java/codenvy-plugin-java-ext-lang-server/src/main/java/com/codenvy/plugin/java/server/WsAgentURLProvider.java]: Can calculate this value on tomcat startup? Is this wsagent side?

**Comment by vparfonov** [comment]
ok

File: plugins/pom.xml [hunk 2]
```diff
@@ -41,5 +41,6 @@
         <module>plugin-microsoft</module>
         <module>plugin-urlfactory</module>
         <module>plugin-product-info</module>
+        <module>../wsagent</module>
```
Comment (skabashnyuk) [inline plugins/pom.xml]: move to parent

**Comment by skabashnyuk** [comment]
other ok

**Pull request merged**